Tiltfile

```diff
@@ -21,5 +21,8 @@ custom_build(
 )
 
 allow_k8s_contexts(k8s_context())
+watch_settings(ignore='config/*')
```

Comment:
why this? I do use the fact that if these files change the kustomize has to be rerun...

Reply:
it is due to the control loop, that goes infinite :) every time a file change (make manifests), even if it is identical.
The first run works, but if you try from the UI to refresh the tiltfile you will see an infinite loop.
Try to comment out and see it.
Kustomize will always run so and will get any changes.